en/simulation/README.md

```diff
@@ -0,0 +1,187 @@
+
+
+### Init File Location
+
+The settings for each configuration target are defined in appropriately named files in [/Firmware/cmake/configs](https://github.com/PX4/Firmware/tree/master/cmake/configs). Within each file there is a setting `config_sitl_rcS_dir` that defines the location of the folder where the configuration stores its init files.
+
+In the cmake config file for [posix_sitl_ekf2](https://github.com/PX4/Firmware/blob/master/cmake/configs/posix_sitl_ekf2.cmake) you can see that the init file will be stored in the folder: **Firmware/posix-configs/SITL/init/ekf2/**.
+```bash
+set(config_sitl_rcS_dir
+    posix-configs/SITL/init/ekf2
+    )
+```    
+
+> **Note** Generally the init files are located using a consistent folder naming convention. For example, `make posix_sitl_ekf2 gazebo_iris` corresponds to the following folder structure:
+```
+Firmware/
+  posix-configs/  (os=posix)
+    SITL/         (platform=sitl)
+      init/       
+        ekf2/     (feature=ekf2)
+          iris    (init file name)
+```
+
+
+### Example Startup File
+
+A slightly reduced version of the startup file for `make posix_sitl_ekf2 gazebo_iris` ([/Firmware/posix-configs/SITL/init/ekf2/iris](https://github.com/PX4/Firmware/blob/master/posix-configs/SITL/init/ekf2/iris)) is shown below.
+
+Note the sections that set parameters, start simulator drivers, and other modules, and that set up MAVLink ports (`mavlink start `) and streaming of messages (`mavlink stream`) to the *QGroundControl* UDP port.
+
+
+```bash
+uorb start
+param load
+dataman start
+param set BAT_N_CELLS 3
+param set CAL_ACC0_ID 1376264
+param set CAL_ACC0_XOFF 0.01
+...
+...
+param set SYS_MC_EST_GROUP 2
+param set SYS_RESTART_TYPE 2
+replay tryapplyparams
+simulator start -s
+tone_alarm start
+gyrosim start
+accelsim start
+barosim start
+adcsim start
+gpssim start
+pwm_out_sim mode_pwm
+sensors start
+commander start
+land_detector start multicopter
+navigator start
+ekf2 start
+mc_pos_control start
+mc_att_control start
+mixer load /dev/pwm_output0 ROMFS/px4fmu_common/mixers/quad_dc.main.mix
+mavlink start -u 14556 -r 4000000
+mavlink start -u 14557 -r 4000000 -m onboard -o 14540
+mavlink stream -r 50 -s POSITION_TARGET_LOCAL_NED -u 14556
+mavlink stream -r 50 -s LOCAL_POSITION_NED -u 14556
+mavlink stream -r 50 -s GLOBAL_POSITION_INT -u 14556
+mavlink stream -r 50 -s ATTITUDE -u 14556
+mavlink stream -r 50 -s ATTITUDE_QUATERNION -u 14556
+mavlink stream -r 50 -s ATTITUDE_TARGET -u 14556
+mavlink stream -r 50 -s SERVO_OUTPUT_RAW_0 -u 14556
+mavlink stream -r 20 -s RC_CHANNELS -u 14556
+mavlink stream -r 250 -s HIGHRES_IMU -u 14556
+mavlink stream -r 10 -s OPTICAL_FLOW_RAD -u 14556
+logger start -e -t
+mavlink boot_complete
+replay trystart
+```
+
+
+## HITL Simulation Environment
```

Comment:
@dagar Awesome. 

 > The specific HIL airframes skip starting the real sensor drivers and then start the sensors and commander modules in HIL mode from the beginning

1.  Is that all done/caused by adding `set HIL yes` to the init file (e.g. as in [1001_rc_quad_x.hil](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/1001_rc_quad_x.hil#L15)
2. Following on from that, if I added `set HIL yes` to another (non hil) init file like [10015_tbs_discovery](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/10015_tbs_discovery) is it now a HIL file - ie can I build the custom file and upload it to firmware and expect it to work like another HIL target, or do I need to specifically enable/disable other things?

> (they can be switched into HIL later on as well).

3. How? I think what your'e saying here is that perhaps I can do something to make a running firmware work in HIL mode. Is this a parameter setting?